### PR TITLE
Shitcuritron

### PR DESCRIPTION
**Pull request by lmenvs**
## About The Pull Request
Adds custom Securitron, fully voiced and new custom sprites.  Currently is a 5% to spawn when constructing a securitron

## Why It's Good For The Game
More things to give the server its own charm

## Changelog
:cl:
add: Added shitcuritron
/:cl:

### CODE REVIEW

File: monkestation/code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	icon_state = "secbot"
+	chasesounds = list('monkestation/sound/voice/pizzky/criminal.ogg','monkestation/sound/voice/pizzky/freeze.ogg','monkestation/sound/voice/pizzky/justice.ogg')
+	arrestsounds = "pizzky"
+	auto_patrol = TRUE
+	verb_whisper = "grumbles"
+	var/last_grumble_speak = 0
+
+/mob/living/simple_animal/bot/secbot/pizzky/bot_patrol()
+	. = ..()
+	if((last_grumble_speak + 1000) < world.time) //these messages should be fairly rare
```
Comment (Zonespace27) [inline monkestation/code/modules/mob/living/simple_animal/bot/secbot.dm:14]:
```suggestion
	if((last_grumble_speak + 100 SECONDS) < world.time) //these messages should be fairly rare
```

File: code/modules/mob/living/simple_animal/bot/construction.dm [hunk 2]
```diff
@@ -390,11 +390,19 @@
...
+					var/mob/living/simple_animal/bot/secbot/S = new(Tsec)
+					S.name = created_name
+					S.baton_type = I.type
+					S.robot_arm = robot_arm
+				else
+					to_chat(user, "<span class='notice'>You complete the... Shitcuritron? Are you sure you did that right?</span>")
+					var/mob/living/simple_animal/bot/secbot/pizzky/S = new(Tsec)
+					S.name = created_name
+					S.baton_type = I.type
+					S.robot_arm = robot_arm
```
Comment (Kapu1178) [inline code/modules/mob/living/simple_animal/bot/construction.dm:404]: Ton of copy-paste. Just put the to_chat in the if/else block nad move everything else out

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/bot/construction.dm (hunk 2 above)
Comment (nednaZ) [reply code/modules/mob/living/simple_animal/bot/construction.dm:404]: That would not work, Pizzky is a separate mob entirely and couldn't be defined before the check

Comment (Merlin1230) [reply code/modules/mob/living/simple_animal/bot/construction.dm:404]: could do something like this:
```dm
var/mob/living/simple_animal/bot/secbot/S
if(prob(95))//monkestation edit
	to_chat(user, "<span class='notice'>You complete the Securitron! Beep boop.</span>")
	S = new(Tsec)
else
	to_chat(user, "<span class='notice'>You complete the... Shitcuritron? Are you sure you did that right?</span>")
	S = new mob/living/simple_animal/bot/secbot/pizzky(Tsec)
S.name = created_name
S.baton_type = I.type
S.robot_arm = robot_arm

**Review by nednaZ — APPROVED** [review]
Looks good to me.

**Pull request merged**